exchanges/orderbook/orderbook.go

```diff
@@ -32,17 +32,17 @@ type Item struct {
 
 // Base holds the fields for the orderbook base
 type Base struct {
-	Pair         pair.CurrencyPair `json:"pair"`
-	CurrencyPair string            `json:"CurrencyPair"`
-	Bids         []Item            `json:"bids"`
-	Asks         []Item            `json:"asks"`
-	LastUpdated  time.Time         `json:"last_updated"`
+	Pair         currency.Pair `json:"pair"`
+	Bids         []Item        `json:"bids"`
+	Asks         []Item        `json:"asks"`
+	lastUpdated  time.Time
```

Comment:
why is lastUpdated changed into an internal field? I create in my own code extra exchanges that implement the IBotExchange interface. With this change, I can not set this field in my code anymore?

Reply:
I used this to limit calls to set this, as this is only needed to be updated when changed. Will re-export it if you are using it in your code other ways.

Reply:
I thought it was meant to store the time  of the last orderbook update from the exchange?

Reply:
Could you please give me points on why we would need a specific field when an Exchange has updated its orderbook, is this ensuring or determining request time delay?

Reply:
I'm using it for okgroup for example. I use the timestamp that okgroup sends to determine whether data sent is before or after other WS messages. Using our own time.Now when updating/setting the orderbook doesn't help in this scenario as it relies on when GCT processes data, not when the exchange does. Making lastupdated inaccessible doesn't allow me to use their timestamp data and so cannot accurately compare orderbook information

Reply:
yeah, me too, I set this field to time.Now() after I receive an orderbook from the exchange after I pulled it using its REST API, so the orderbook has a timestamp when I process it further

Reply:
OKay will revert and export time value and do a check to see if its set if not; time.Now() Thanks for your input guys!